modules/claims_api/app/swagger/claims_api/description/v0.md

```diff
@@ -12,7 +12,6 @@ Currently, the API supports automatic establishment of:
 
  - Application for Disability Compensation (21-526EZ)
  - Intent to File (21-0966)
- - Appointment of Representative (21-22 and 21-22a) **(in test environments only)**
```

Comment:
here the whole line is struck, but in v1 only the part about being in test is removed.  Is this correct?

Reply:
@erluti yes that is correct per my conversation with Kelly Lein due to power of attorney endpoints not being in docs on v0 all-together.

Reply:
We've tested V1 but due to the internal nature of V0 and is not Oauth based then there's still some work for us to do to successfully launch it in V0 because of BGS constraints.